src/utils/FontUtils.ts

```
@@ -27,10 +27,12 @@ import {
   LineJoinStyle,
   setFillingColor,
   setStrokingColor,
+  rgb,
```

ESLint

[error] 'rgb' is defined but never used.

(@typescript-eslint/no-unused-vars)